Source/CompanyCommunicator.Common/Configuration/DODConfiguration.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Microsoft.Teams.Apps.CompanyCommunicator.Common.Configuration
+{
+    /// <summary>
+    /// App configuration for DOD environment.
+    /// </summary>
+    internal class DODConfiguration : IAppConfiguration
+    {
+        private readonly string tenantId;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DODConfiguration"/> class.
+        /// </summary>
+        /// <param name="tenantId">Tenant Id.</param>
+        public DODConfiguration(string tenantId)
+        {
+            this.tenantId = tenantId ?? throw new System.ArgumentNullException(nameof(tenantId));
+        }
+
+        /// <inheritdoc/>
+        public string AzureAd_Instance => "https://login.microsoftonline.us/";
+
+        /// <inheritdoc/>
+        public string AzureAd_ValidIssuers => $"https://login.microsoftonline.us/{this.tenantId}/v2.0,https://sts.windows.net/{this.tenantId}/";
+
+        /// <inheritdoc/>
+        public string AuthorityUri => $"https://login.microsoftonline.us/{this.tenantId}";
+
+        /// <inheritdoc/>
+        public string GraphBaseUrl => "https://graph.microsoft.us/v1.0";
```

Comment:
Is this correct? The docs say https://dod-graph.microsoft.us (https://learn.microsoft.com/en-us/graph/deployments)

Reply:
it should be "dod-graph". Thanks for catching that.